raft/raft.go

```diff
@@ -31,6 +31,8 @@ const noLimit = math.MaxUint64
 
 var errNoLeader = errors.New("no leader")
 
+var ErrTemporarilyUnavailable = errors.New("snapshot is temporarily unavailable")
```

Comment:
Either name the variable `ErrSnapshotTemporarilyUnavailable` or remove the word "snapshot" from the message.